ExtendedDescription.md

```diff
@@ -0,0 +1,130 @@
+## About nelson.social
+
+Welcome to nelson.social, an online space dedicated to the community of Nelson, BC! We’re excited to build a place where people from all walks of life can come together to share ideas and passions, build connections, or just hang out and chat.
+
+Here at nelson.social, we believe in the power of community and the importance of supporting one another in order to enrich our community. Whether you are a lifelong resident of Nelson, new to the region, or just passing through, we invite you to join the conversation and be a part of our community.
+
+In this space, you can share your thoughts and ideas, ask questions of your neighbours, and even plan events and activities. You can also connect with people from all over the world, because nelson.social is part of the [Mastodon](https://github.com/mastodon/) network. We like to think of it like a big WhatsApp or iMessage group, but where you can choose who to share with and whose messages you see, even if they aren’t members of nelson.social.
+
+We encourage you to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. We encourage you to read them, and also reach out to us if you encounter any problems, or have any questions.
+
+We look forward to seeing you on nelson.social, and to building a vibrant and supportive community together. Welcome!
+
+## Land Acknowledgement
+
+We would like to acknowledge that in Nelson we live and work on the traditional, ancestral and unceded territory of Sinixt Peoples. We make this acknowledgment to show our respect for the tmxʷulaʔxʷ (homeland), and [Sinixt](https://bloodoflifecollective.org/sinixt.html) Nation. We feel privileged and grateful to be here, and we invite those reading this to explore their own relationship to place.
+
+## Code of Conduct
+
+Nelson is home to people from all over the world who have a diverse set of skills, personalities, backgrounds and experiences. This diversity is wonderful, and we are committed to ensuring that the nelson.social community is a safe, enjoyable and thriving environment for the largest number possible of Nelson residents, their friends and family members.
+
+To that end, we will strive to make this a friendly, safe and welcoming environment for everyone regardless of qualities such as their gender identity or expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, nationality, ethnicity, socioeconomic status, family structure, spirituality, religion (or lack thereof), education, or other personal traits. We particularly celebrate diversity and do not tolerate bigotry or prejudice.
```

Comment:
Love this.

Genuine question: Does it mean I can't joke about people who believe the earth is flat or that they are a mermaid with x-ray vision?

Reply:
I think that would be absolutely fine. Which of the traits on that list would you be worried that would come under? Religion?! 😀

Reply:
I'm really just curious what radical inclusivity looks like in the real world. Maybe better to be discussed over a cuppa 🇬🇧